server/__init__.py

```diff
@@ -371,7 +371,7 @@ def create_app(nl_root=DEFAULT_NL_ROOT):
         secret_response = secret_client.access_secret_version(name=secret_name)
         app.config['PALM_API_KEY'] = secret_response.payload.data.decode(
             'UTF-8')
-    app.config['NL_BAD_WORDS'] = bad_words.load_bad_words()
+    app.config['NL_BAD_WORDS'] = bad_words.load_bad_words(load=not cfg.CUSTOM)
```

Comment:
Should we be consistent with use of `flask_env == 'custom'` vs. `cf.CUSTOM`?

Reply:
We should have some one line boolean helper function for such checks?

Reply:
I went by how it was used elsewhere in the file.

Would like to understand the difference though. As I understand it, `flask_env` is read from the env whereas cfg.CUSTOM is a `bool` in the config. What happens if both are different?

Reply:
flask_env could be "floret", "stanford" or "custom" but they all have cfg.CUSTOM set. I guess here we are really looking at a new cfg field like "BLOCK_BAD_WORDS"..?

Reply:
Ah ok. In that case, we are currently not loading banned words for all custom DCs so cfg.CUSTOM is appropriate.

However, we'll need to change the logic in the NL loader.

The `get_env_path` method there uses the env variable. However, we need it to pick the `deploy/nl` path for all custom DCs.

@pradh - do we have access to the config in the loader?

Reply:
Oh I think the `cfg` thing is a website server thing, not in NL server I think.

Reply:
Hmm. In nl server, will `flask_env` be `custom` for all custom DCs or will it be `floret`, `stanford`, etc?

Reply:
flask_env could be floret, stanford, this env is global for all the servers

Reply:
In that case, should we flip the logic like below?

```
def get_env_path(flask_env: str, file_name: str) -> str:
  if flask_env in ['autopush', 'staging', 'production']:
    return return f'/datacommons/nl/{file_name}'

  return os.path.join(
        os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
        f'deploy/nl/{file_name}')
```

Reply:
I'm closing this conversation and opening another one in the affected file.